src/code/code_n64dd_800AD4C0.c

```diff
@@ -1,3 +1,90 @@
+#include "global.h"
 #include "n64dd.h"
 
+// TODO functions of unknown prototype
+extern char func_801C8510_unknown[];
+extern char func_800AE170_unknown[];
+extern char func_800ADCD8_unknown[];
```

Comment:
Seems like 800AE170 is `Fault_DrawText` and `800ADCD8` is `Fault_WaitForInput` from your other PR. Not sure how you want to handle it but it seems fine for now

Reply:
yeah I noticed, I want to handle it by waiting for either PR to be merged and then updating the other